src/composables/useSubscription.js      |   5 ++
1 file changed, 5 insertions(+)

--- a/src/composables/useSubscription.js
+++ b/src/composables/useSubscription.js
@@ -49,6 +49,11 @@ export function useSubscription() {
       if (form.tpl.clash.doh === true) {
         params += "&clash.doh=true";
       }
+
+      if (!form.expand) {
+        params += "&classic=true";
+      }
+
       params += "&new_name=" + form.new_name.toString();
     }
 