--- a/src/config.ts
+++ b/src/config.ts
@@ -56,5 +56,5 @@ export const backendConfig = [{
-    label: 'localhost:25500(自建本地服务)',
-    value: 'http://localhost:25500/sub?'
+    label: `${window.location.protocol}//${window.location.hostname}:25500(自建本地服务)`,
+    value: `${window.location.protocol}//${window.location.hostname}:25500/sub?`
 }];
 export const externalConfig = [{
     label: 'Custom规则',
